cmd/juju/status/formatted.go

```diff
@@ -43,6 +42,11 @@ type modelStatus struct {
 	Status           statusInfoContents `json:"model-status,omitempty" yaml:"model-status,omitempty"`
 	MeterStatus      *meterStatus       `json:"meter-status,omitempty" yaml:"meter-status,omitempty"`
 	SLA              string             `json:"sla,omitempty" yaml:"sla,omitempty"`
+	ControllerStatus *controllerStatus  `json:"controller-status,omitempty" yaml:"controller-status,omitempty"`
```

Comment:
why here vs the root level we talked about originally? Was there a limitation that you hit?

Reply:
Ah, sure I'll move it to there no problem. I was thinking from a consistency level that tabular is model and so should json and yaml. That's my bad.